line_h_up: 62
ball_v_low: 62
ball_h_low: 24
ball_s_low: 164
ball_s_up: 255
field_s_up: 255
ball_h_up: 37
field_v_low: 144
field_h_low: 43
ball_v_up: 255
field_s_low: 28
field_h_up: 82
line_s_low: 0
line_h_low: 8
field_v_up: 255
line_v_low: 197
line_s_up: 60
line_v_up: 255